governance/votes/Executive vote - August 22, 2024.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Template - [Executive Vote] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024
+summary: Update PSM State Variable in Conduit contracts to MCD_LITE_PSM_USDC_A, USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A, Update PSM-USDC-A Fees, Update PSM-USDC-A DC-IAM, Update MCD_LITE_PSM_USDC_A buf, Update LITE-PSM-USDC-A DC-IAM, GSM Delay Update, Update LitePSM Keeper Network Job.
+date: 2024-08-22T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] LITE-PSM-USDC-A Phase 2 Setup - August 22, 2024
+
+The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
+
+- The PSM state variables in the conduit contracts will be updated to the MCD_LITE_PSM_USDC_A.
+- **200 million USDC** reserves in PSM-USDC-A will be migrated to LITE-PSM-USDC-A.
```

Comment:
This is not the proposed action - https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839

> Migrate all but 200M USDC reserves from PSM-USDC-A to LitePSM with a script executed in the spell (has to be done before PSM-USDC-A tin & tout is increased in the spell transaction)